tests/Functional/ViewHelpers/JoinViewHelperTest .php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3\CMS\Fluid\Tests\Functional\ViewHelpers;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Test;
+use TYPO3Fluid\Fluid\Tests\Functional\AbstractFunctionalTestCase;
+use TYPO3Fluid\Fluid\View\TemplateView;
+
+final class JoinViewHelperTest extends AbstractFunctionalTestCase
+{
+    public static function renderValidDataProvider(): \Generator
+    {
+        yield 'template with named layout' => [
+            'arguments' => [
+                'value' => [],
+            ],
+            'src' => '<f:join value="{value}" />',
+            'expectation' => '',
+        ];
+        yield 'empty value inline' => [
+            'arguments' => [
+                'value' => [],
+            ],
+            'src' => '{value -> f:join()}',
+            'expectation' => '',
+        ];
+        yield 'empty attribute value, with separator' => [
+            'arguments' => [
+                'value' => [],
+                'separator' => ',',
+            ],
+            'src' => '<f:join value="{value}" separator="{separator}" />',
+            'expectation' => '',
+        ];
+        yield 'empty attribute value, with separatorLast' => [
+            'arguments' => [
+                'value' => [],
+                'separatorLast' => ' and ',
+            ],
+            'src' => '<f:join value="{value}" separatorLast="{separatorLast}" />',
+            'expectation' => '',
+        ];
+        yield 'value attribute' => [
+            'arguments' => [
+                'value' => [1, 2, 3],
+            ],
+            'src' => '<f:join value="{value}" />',
+            'expectation' => '123',
+        ];
+        yield 'value attribute inline array' => [
+            'arguments' => [
+                'value' => [1, 2, 3],
+            ],
+            'src' => '<f:join value="{0: \'1\', 1: \'2\', 2: \'3\'}" />',
+            'expectation' => '123',
+        ];
+        yield 'value inline' => [
```

Comment:
Can you add another case where both inline and argument are used to test the fallback order?